test_suite/conftest.py

```diff
@@ -10,6 +11,11 @@
 
 @pytest.fixture(autouse=True, scope='function')
 def check_markers(host, request):
+    skip_massage = "This test doesn't apply to {distro_version}"
+
+    def parse_distro_version(distro_version_list):
```

Comment:
I think the name of this arg is misleading. We are not passing a list, but a single item which is a string containing distro, major version and minor version. E.g. "rhel9.1".

Reply:
You're right:)
will fix